src/components/Tooltip/Tooltip.vue

```diff
@@ -23,48 +29,60 @@
         props: {
             label: {
                 type: String,
-                required: true
             },
             placement: {
                 type: String,
-                default: 'bottom'
+                default: 'bottom',
             },
             disabled: {
                 type: Boolean,
-                default: false
+                default: false,
             },
             offset: {
                 type: String,
-                default: '0,5'
+                default: '0,5',
             },
             appendToBody: {
                 type: Boolean,
-                default: false
+                default: false,
             },
             boundariesElement: {
                 type: String,
-                default: 'viewport'
-            }
+                default: 'viewport',
+            },
+            delay: {
+                type: Number,
+                default: 0,
+            },
         },
         data() {
             return {
-                show: false
+                mouseEntered: false,
+                show: false,
             };
         },
         methods: {
             onMouseEnter() {
-                this.show = true;
-                if (this.appendToBody) {
-                    this.$nextTick(() => {
-                        this.append();
-                    });
-                }
+                this.mouseEntered = true;
+                setTimeout(() => {
+                    if (!this.mouseEntered) return;
```

Comment:
This is a problem only for bigger delays so maybe we can ignore it.
You can for example for 5s delay:
1. hover over the button
2. wait 1s
3. hover away
4. wait 3s
5. hover back on
6. wait 1s
Result: tooltip opens up after 1s from hovering on.
You can get similar effects for closing the tooltip.

Reply:
fixed